core/util.go

```diff
@@ -262,6 +263,17 @@ func LoadCert(filename string) (cert *x509.Certificate, err error) {
 	return
 }
 
+// LoadJWK loads a JSON encoded JWK specified by filename or returns an error
```

Comment:
We talked OOB: consensus was not adding to `core` is a good call :+1: but that rather than making a new `jwk` package for one function it was acceptable to add it to `web`. I've done that in 
3e1a168

Reply:
(And in c4f4d21 because I forgot to `git add` the new `.go` file :blush:)